go-controller/pkg/ovn/gateway_test.go

```diff
@@ -30,8 +32,14 @@ var (
 	nodeName = "test-node"
 )
 
+func init() {
+	// libovsdb matcher might produce a lengthy output that will be cropped by
+	// default gomega output limit, set to 0 to unlimit.
+	format.MaxLength = 0
+}
```

Comment:
you should be able to remove this. 
https://github.com/ovn-org/ovn-kubernetes/blob/bc2a572c39baa0d9fab91af334239551de4484ad/go-controller/pkg/testing/libovsdb/matchers.go#L14-L18

Reply:
ack. done